bin/eslint

```diff
@@ -37,6 +37,11 @@ if (process.env.ENGINE_CONFIG) {
   if (engineConfig['exclude_paths']) {
     ignores = engineConfig['exclude_paths'];
   }
+  if (engineConfig['extensions']) {
+    options['extensions'] = engineConfig['extensions'];
+  } else {
+    options['extensions'] = ['.js'];
```

Comment:
You can leave this above, where it was. With the change as it is, if there is no engine config, we will not set an extensions option